launcher/minecraft/mod/ResourcePack.cpp

```diff
@@ -17,15 +17,17 @@ static const QMap<int, std::pair<Version, Version>> s_pack_format_versions = {
     { 3, { Version("1.11"), Version("1.12.2") } }, { 4, { Version("1.13"), Version("1.14.4") } },
     { 5, { Version("1.15"), Version("1.16.1") } }, { 6, { Version("1.16.2"), Version("1.16.5") } },
     { 7, { Version("1.17"), Version("1.17.1") } }, { 8, { Version("1.18"), Version("1.18.2") } },
-    { 9, { Version("1.19"), Version("1.19.2") } }, { 11, { Version("1.19.3"), Version("1.19.3") } },
+    { 9, { Version("1.19"), Version("1.19.2") } },
+    // { 11, { Version("22w42a"), Version("22w44a") } }
```

Comment:
i guess this should be uncommented?

Reply:
will it cause problems with the non numeric version?

Reply:
i _think_ not, iirc this is only used for display, and in the filtering, where it _should_ be fine since it only converts directly to a string :thinking: